packages/cachyos-ananicy-rules-git/cachyos-ananicy-rules-git.pacscript

```diff
@@ -0,0 +1,18 @@
+name="cachyos-ananicy-rules-git"
+gives="cachyos-ananicy-rules"
+pkgver="0.0.1"
+pkgdesc="ananicy-cpp rules for CachyOS"
+pacdeps=('ananicy-cpp')
+homepage="https://github.com/CachyOS/ananicy-rules"
+url="https://github.com/CachyOS/ananicy-rules.git"
+maintainer="vigress8 <vig@disroot.org>"
+pkgver() {
+  git ls-remote "${url}" master | cut -f1 | cut -c1-8
+}
+
+package() {
+  sudo install -Dm644 LICENSE -t "${pkgdir}/usr/share/licenses/${gives}"
+  rm -f README.md LICENSE
+  sudo install -d "${pkgdir}/etc/ananicy.d"
+  sudo cp -rf ./* "${pkgdir}/etc/ananicy.d"
```

Comment:
You can combine these two into one `install` statement so let's do that.

Reply:
`install` can't preserve modes, can it? So copying non-executable files and directories at once is problematic

Reply:
> `install` can't preserve modes, can it? So copying non-executable files and directories at once is problematic

The `-Dm` flag can preserve file permissions if that's what you mean.

Reply:
```
-m, --mode=MODE
set permission mode (as in chmod), instead of rwxr-xr-x
```

So it uses 755 by default, but you may change the mode. Problem is, it applies the mode to everything it copies, and directories need to be executable